src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -67,17 +68,21 @@ def _check_init_scripts(self, cluster, failures):
                 continue
             failures.append(f"{_AZURE_SP_CONF_FAILURE_MSG} cluster.")
 
-    def _check_cluster_failures(self, cluster: ClusterDetails):
+    def _check_cluster_failures(self, cluster: ClusterDetails | compute.ClusterSpec):
```

Comment:
This is irrelevant to this PR, separate it

Reply:
Created PR #839

Reply:
Done